packages/autocomplete-core/src/utils/__tests__/isAndroid.test.ts

```diff
@@ -0,0 +1,293 @@
+import { isAndroid } from '../isAndroid';
```

Comment:
I'm wondering if we need to check on Android devices since it seems that [all Samsung phones run on Android](https://www.samsung.com/uk/support/mobile-devices/how-can-i-check-what-version-of-android-i-have-on-my-device/). If tests are still passing without it, we should likely remove it to bring the bundle size down. wdyt?

Reply:
Could a Samsung windows computer declare itself as Samsung, or does that not happen? in any case, that wouldn't have a big impact, as the enterkeyhint only has effect on mobile devices (which don't have arrow keys, so I still don't get how you could ever see something else than search)

Reply:
> I'm wondering if we need to check on Android devices since it seems that [all Samsung phones run on Android](https://www.samsung.com/uk/support/mobile-devices/how-can-i-check-what-version-of-android-i-have-on-my-device/). If tests are still passing without it, we should likely remove it to bring the bundle size down. wdyt?

Indeed, we could skip this check, but Samsung desktop on Chrome will have a static `enterKeyHint`, but as @Haroenv mentioned, it should not be an issue since this property is only use by virtual keyboards on mobile devices, WDYT?

> Could a Samsung windows computer declare itself as Samsung, or does that not happen? in any case, that wouldn't have a big impact, as the enterkeyhint only has effect on mobile devices (which don't have arrow keys, so I still don't get how you could ever see something else than search)

I found an user agent for the Samsung Chromebook and yes, it's flag as Samsung, so `isSamsung('Mozilla/5.0 (Linux; Android 9; Samsung Chromebook Plus (V2) Build/R94-14150.64.0; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/96.0.4664.45 Safari/537.36') = true`

Reply:
I think we can go ahead in and drop `isAndroid` in this case, which will keep the bundle size tiny.